includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -399,6 +418,7 @@ public function __construct( DOMDocument $dom, array $args = [] ) {
 		$this->base_url    = untrailingslashit( $guessurl );
 		$this->content_url = WP_CONTENT_URL;
 		$this->xpath       = new DOMXPath( $dom );
+		$this->cache_pool  = new AMP_Cache_Pool( self::CACHE_GROUP );
```

Comment:
Perhaps the pool size should be passed as an argument from `$this->args`? If a default were supplied in `$DEFAULT_ARGS` then this would allow for a site to override the default if the default is not suitable.

```suggestion
		$this->cache_pool  = new AMP_Cache_Pool( self::CACHE_GROUP, isset( $this->args['cache_pool_size'] ? $this->args['cache_pool_size'] : AMP_Cache_Pool::DEFAULT_POOL_SIZE ) );
```

Reply:
Yes, that might come in handy. I can't commit your suggestion as is, though, it is broken (parentheses in the wrong position).

Reply:
Fixed 😊